tests/unit/test_column_noise.py

```diff
@@ -182,7 +182,6 @@ def test_generate_within_household_copies():
 
 
 def test_swap_months_and_days(dummy_dataset):
-
```

Comment:
Why? Are you doing black to the whole repo?

Reply:
I always do black to prevent build failing. Not sure why this happened.